lib/sentry/opentelemetry/span_storage.ex

```diff
@@ -0,0 +1,132 @@
+defmodule Sentry.OpenTelemetry.SpanStorage do
+  @moduledoc false
+  use GenServer
+
+  @table :span_storage
```

Comment:
This is confusing to me. If we allow the server to be named whatever, but then we enforce the name for the ETS table, then we allow multiple servers to interact with the same table, which is not what we want I think.

Reply:
@whatyouhide right, I forgot that this was the original idea :/ could you give me some hints how to best implement configurable table names though?